modules/govuk_jenkins/manifests/jobs/athena_fastly_logs_check.pp

```diff
@@ -0,0 +1,48 @@
+# == Class: govuk_jenkins::jobs::athena_fastly_logs_check
+#
+# Create a file on disk that can be parsed by jenkins-job-builder
+#
+# === Parameters
+#
+# [*aws_access_key_id*]
+#   Access key id for user who can read from Athena and write to the bucket
+#
+# [*aws_secret_access_key*]
+#   Secret access key for the above user
+#
+# [*aws_region*]
+#   The region athena is running in
+#
+# [*s3_results_bucket*]
+#   An S3 bucket where the query results can be written too
+#
+class govuk_jenkins::jobs::athena_fastly_logs_check (
+  $aws_access_key_id = undef,
+  $aws_secret_access_key = undef,
+  $aws_region = 'eu-west-1',
+  $databases = ['govuk_www', 'govuk_assets'],
```

Comment:
Nitpick: This value should probably be added the parameters list in the comment.

Reply:
Thank you! sorted